app/main.py

```diff
@@ -1 +1,15 @@
-# write your code here
+def calculate_team_total_rating(players: list):
+    calc_rate = 0
+    for player in players:
+        calc_rate += player.get_rating()
+    return calc_rate + 3
```

Comment:
why did you add 3?

Reply:
![image](https://user-images.githubusercontent.com/110998159/190125128-692e1aff-a108-4364-a86b-49e7f513991d.png)

i saw that at the example.